### PR TITLE
AG-6646 - Clean up of charts overview and getting started docs pages

### PR DESCRIPTION
**Pull request by manapeirov**

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-getting-started/index.md [hunk 1]
```diff
@@ -409,6 +409,10 @@ AG Charts is an exciting new addition to the AG Grid family, offering both integ
 </div>
 </section>
 
+[[only-vue]]
+| [[note]]
+| | The "Quick Look Code" above is different to the code that runs in Stackblitz - this is because Stackblitz doesn't support .vue files at this time.
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-getting-started/index.md:414]: Nice addition - I wasn't aware of this :-)

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-getting-started/index.md [hunk 2]
```diff
@@ -548,7 +552,7 @@ AG Charts is an exciting new addition to the AG Grid family, offering both integ
 |
 | Here we'll provide the `options` we want to use for our chart, including the `series` to use to plot the data.
 |
-| The series `type` defaults to `'line'`, so the only series configuration we need to specify is to tell the series which keys to use to fetch the data to be plotted along the horizontal (x) and | vertical (y) axes.
+| The series `type` defaults to `'line'`, so the only series configuration we need to specify is to tell the series which keys to use to fetch the data to be plotted along the horizontal (x) and vertical (y) axes.
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-getting-started/index.md:555]: I see there are several of these issues - did you analyse the root cause? (My main concern is: did a tool break the formatting, and if so should we be avoiding performing auto-reformatting on `.md` files with `|` sections?)

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/index.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
...
+Our standalone chart library, AG Charts, is flexible and powerful, enabling creation of charts independently of the grid.
 
-As with the grid, the "ag" part of AG Charts stands for "agnostic". The internal AG Charts engine is implemented in TypeScript with zero dependencies. You can use the vanilla JavaScript AG Charts library, or alternatively take advantage of the framework-specific AG Charts Components to integrate with any major framework.
+To jump straight in, click through to the [Getting Started](/charts-getting-started/) section, or head to the [API Explorer](/charts-api-explorer/). Alternatively, scroll down for a gallery of examples demonstrating a variety of visualisations created using our library.
 
-<chart-gallery></chart-gallery>
+As with the grid, the "AG" part of AG Charts stands for "agnostic". The internal AG Charts engine is implemented in TypeScript with zero dependencies. Framework specific AG Charts components allow integration with all major front-end frameworks used to create enterprise applications.
+
+<chart-gallery></chart-gallery>x
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/index.md:11]: Nit: trailing `x` on this line

**Review by alantreadway — COMMENTED** [review]
One nit that needs addressing, but generally LGTM :-)

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/index.md (hunk 3 above)
Comment (manapeirov) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-overview/index.md]: ah yes good spot

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**